src/ru/skypro/Main.java

```diff
@@ -1,8 +1,44 @@
 package ru.skypro;
 
 public class Main {
-
     public static void main(String[] args) {
-	// write your code here
+        Task1();
+        Task2();
+        Task3();
+    }
+    public static void Task1()
```

Comment:
методы именуются с прописной бувы
```suggestion
    public static void task1()
```